core/components/css-utilities/Overflow/Overflow.story.tsx

```diff
@@ -12,62 +12,318 @@ export const overflow = () => {
-      &nbsp;
-      <Heading size="s">The auto value adds scrollbars when necessary:</Heading>
-      <br />
-      <div className="overflow-auto Utilities-overflow">
-        You can use the overflow property when you want to have better control of the layout. The overflow property
-        specifies what happens if content overflows an element's box.
-      </div>
-      &nbsp;
-      <div className="DocPage-codeBlock">
-        {'<div className="overflow-auto overflow">'}
-        <br />
-        {
-          'You can use the overflow property when you want to have better control of the layout. The overflow property specifies what happens if content overflows an element box.'
-        }
-        <br />
-        {'</div>'}
-      </div>
-      &nbsp;
-      <Heading size="s">With the hidden value, the overflow is clipped, and the rest of the content is hidden</Heading>
-      <br />
-      <div className="overflow-hidden Utilities-overflow">
-        You can use the overflow property when you want to have better control of the layout. The overflow property
-        specifies what happens if content overflows an element's box.
-      </div>
-      &nbsp;
-      <div className="DocPage-codeBlock">
-        {'<div className="overflow-hidden overflow">'}
-        <br />
-        {
-          'You can use the overflow property when you want to have better control of the layout. The overflow property specifies what happens if content overflows an elements box.'
-        }
-        <br />
-        {'</div>'}
+
+      <div className="mt-9">
+        <Heading size="m">Examples</Heading>
+        <Text className="mb-6">
+          Here's how each overflow utility affects content that exceeds its container's dimensions:
+        </Text>
+
+        {/* General Overflow Examples */}
+        <Card className="mb-8 p-6">
+          <Heading size="s">General Overflow</Heading>
+          <Text className="mt-3 mb-5">These utilities control overall overflow in both directions.</Text>
+
+          <div className="d-flex flex-wrap">
+            {/* overflow-auto */}
+            <div className="mr-8 mb-6 w-25">
+              <Text weight="strong" className="mb-3">
+                overflow-auto
+              </Text>
+              <div className="position-relative">
+                <div className="overflow-auto p-4 bg-secondary-lightest border rounded-05 h-100">{longText}</div>
+                <Text className="mt-3">Scrollbars appear only when necessary</Text>
+              </div>
+            </div>
+
+            {/* overflow-hidden */}
+            <div className="mr-8 mb-6 w-25">
+              <Text weight="strong" className="mb-3">
+                overflow-hidden
+              </Text>
+              <div className="position-relative">
+                <div className="overflow-hidden p-4 bg-secondary-lightest border rounded-05 h-100">{longText}</div>
+                <Text className="mt-3">Content is clipped, no scrollbars</Text>
```

Comment:
Please use some better example, right now all the examples looks similar 
<img width="928" alt="image" src="https://github.com/user-attachments/assets/3a2551b8-a1d3-48f7-962b-4149a0dc4aff" />